### ISSUE TITLE
REPORT request needs 'prop' entry

### ISSUE THREAD

**Issue by individual-it** [owncloud/docs#447]
## What Type Of Content Change Is This?

- [ ] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [x] Bug Fix to Existing Content

## Which Manual Does This Relate To?

- [ ] Admin Manual
- [x] Developer Manual
- [ ] User Manual

## What Needs to be Documented?
When doing a REPORT to list favored files and folders as described in the documentation https://doc.owncloud.com/server/developer_manual/webdav_api/search.html#example-request-bodies-2
`curl -u admin:admin -X REPORT http://localhost/owncloud-core/remote.php/dav/files/admin -d '<?xml version="1.0" encoding="UTF-8"?> <oc:filter-files xmlns:a="DAV:" xmlns:oc="http://owncloud.org/ns">     <oc:filter-rules>         <oc:favorite>1</oc:favorite>     </oc:filter-rules> </oc:filter-files>' | xmllint --format -`
I'm receiving
`<d:status>HTTP/1.1 418 I'm a teapot</d:status>`

adding `<a:prop><oc:favorite/></a:prop>` gives `<d:status>HTTP/1.1 200 OK</d:status>`

## Why Should This Change Be Made?
because `HTTP/1.1 418 I'm a teapot` is funny but not useful

**Comment by settermjd** [comment]
Thanks for raising an issue about this, @individual-it.

**Comment by settermjd** [comment]
@individual-it, do you have any feedback on this, or can it now be closed?

**Comment by individual-it** [comment]
still same issue

with 
```
<?xml version="1.0" encoding="UTF-8"?>
<oc:filter-files xmlns:a="DAV:" xmlns:oc="http://owncloud.org/ns">
    <oc:filter-rules>
        <oc:favorite>1</oc:favorite>
    </oc:filter-rules>
</oc:filter-files>
```

I receive `HTTP/1.1 418 I'm a teapot`

with 
```
<?xml version="1.0" encoding="UTF-8"?>
<oc:filter-files xmlns:a="DAV:" xmlns:oc="http://owncloud.org/ns">
    <oc:filter-rules>
        <oc:favorite>1</oc:favorite>
    </oc:filter-rules>
    <a:prop>
        <oc:favorite/>
    </a:prop>
</oc:filter-files>
```
I get `HTTP/1.1 200 OK`

**Comment by settermjd** [comment]
Then there's something different between your environment and mine. Let's see if we can work it out. The installation that I'm using is ownCloud 10.3.0 alpha2, based on the latest version of master. What version are you using?

**Comment by individual-it** [comment]
tried with current master and the released 10.3.0

**Comment by individual-it** [comment]
as this is actually a bug I opened an issue in core https://github.com/owncloud/core/issues/38463
